game/resource/English/ability/items/tooltip_drums_of_endurance_oaa.txt

```diff
@@ -0,0 +1,53 @@
+//===================================================================
+// Drums of Endurance
+//===================================================================
+
+"DOTA_Tooltip_Ability_item_ancient_janggo_1_note2"                        "Drums of Endurance will not use or consume charges from Level 2 onwards"
+
+"DOTA_Tooltip_Ability_item_drums_of_endurance_2"                          "#{DOTA_Tooltip_ability_item_ancient_janggo}"
+"DOTA_Tooltip_Ability_item_recipe_drums_of_endurance_2"                   "#{DOTA_Tooltip_Ability_item_recipe_ancient_janggo}"
+"DOTA_Tooltip_Ability_item_drums_of_endurance_2_description"              "#{DOTA_Tooltip_ability_item_ancient_janggo_Description}"
+"DOTA_Tooltip_Ability_item_drums_of_endurance_2_lore"                     "#{DOTA_Tooltip_ability_item_ancient_janggo_Lore}"
+"DOTA_Tooltip_Ability_item_drums_of_endurance_2_note0"                    "#{DOTA_Tooltip_ability_item_ancient_janggo_Note0}"
+"DOTA_Tooltip_Ability_item_drums_of_endurance_2_note1"                    "#{DOTA_Tooltip_ability_item_ancient_janggo_Note1}"
+"DOTA_Tooltip_Ability_item_drums_of_endurance_2_note2"                    "Drums of Endurance will not use charges from Level 2 onwards"
```

Comment:
This should use a reference as well.

Reply:
References can be made to custom tooltip keys as well. So just reference the note for level 1.

Reply:
Looks like research on moddota website is in order

Reply:
The referencing system is part of the tooling that Chris developed for OAA. Probably won't find anything on ModDota about it. What I mean is that since there is the key `DOTA_Tooltip_Ability_item_ancient_janggo_1_note2` and you want the notes for all the other levels to say the exact same thing, you can just reference that key, i.e. `#{DOTA_Tooltip_Ability_item_ancient_janggo_1_note2}` as the value for the other notes.

Reply:
Aaaahh. Got it. Thanks!